CRM/Odoosync/Form/Configurations.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+use CRM_Odoosync_ExtensionUtil as E;
+
+/**
+ * Odoo Civicrm Sync Configuration form controller
+ *
+ * @see http://wiki.civicrm.org/confluence/display/CRMDOC43/QuickForm+Reference
+ */
+class CRM_Odoosync_Form_Configurations extends CRM_Core_Form {
+
+  /**
+   * Contains array of fields, which config Odoo Civicrm Sync Expansion
+   *
+   * @var string[]
+   */
+  private $settingFields = [];
+
+  /**
+   * Builds the form object.
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  public function buildQuickForm() {
+    $configElements = [];
+    CRM_Utils_System::setTitle(E::ts('CiviCRM Odoo Sync Configuration'));
+
+    $settingFields  = $this->getSettingFields();
```

Comment:
I see that newspaper like order for methods is easier to read, so in this case, getSettingFields() method should be moved right below buildQuickForm() . this apply to all this class methods.